hoot-core/src/main/cpp/hoot/core/io/ApiDb.h

```diff
@@ -190,6 +190,18 @@ class ApiDb
    */
   QSqlDatabase getDB() { return _db; }
 
+  static long round(double x);
+  static long round(double x, int precision);
```

Comment:
Could you add an explanation of the precision parameter in a comment? Also, my eyeball compiler thinks you'll see a divide-by-zero in this function if precision=1... that might be worth a note in a comment too.